src/lib/yacc/grammar.rs

```diff
@@ -1035,4 +1136,42 @@ mod test {
         find("C", vec![vec!["x"]]);
         find("D", vec![vec!["y", "x"], vec!["y", "z"]]);
     }
+
+    #[test]
+    fn test_nonterm_max_costs1() {
+        let grm = yacc_grm(YaccKind::Original, &"
```

Comment:
Why `&"...".to_string()`? Shouldn't just `"..."` work, since it's already an `&str`?

Reply:
I admit, I copy and pasted these...